### ISSUE TITLE
Atmosphere 0.10.0

### ISSUE THREAD

**Issue by gtortone** [noahc3/SDSetup#140]
Hi,
do you plan to update package selection with new Hekate (5.1.0) and
new Atmosphere (0.10.0) ?

Thanks,
Gennaro

**Comment by noahc3** [comment]
SDSetup will automatically update when Kosmos updates, within 12 hours.

**Comment by butiuae** [comment]
Sorry to be a bother but SD setup is not updating automatically

**Comment by hax4dazy** [comment]
`SDSetup will automatically update when Kosmos updates`

Hmm i wonder why.

**Comment by NicholeMattera** [comment]
We are still waiting for a few Homebrew to update before releasing v14.3 of Kosmos. Once that is released then within 12 hours SDSetup will be updated.

**Comment by GizmoTheGreen** [comment]
well thats great because kosmos updater did get something (I didn't check here first) and assumed it would be the update for new FW release so I let it run and left me in an unbootable state for CFW, so only stock works. (weird combo of atmosphere and old hekate I guess?) so updated hekate myself and still couldn't boot, failed on sigpatch but I never use that since I don't pirate???

TL;DR updater is doing weird shit with cfw setups
